dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -204,12 +204,14 @@
 		status = "disabled";
 	};
 
-	flexcomm8: flexcomm@9f000 {
-		compatible = "nxp,lpc-flexcomm";
+	hs_lspi: spi@9f000 {
```

Comment:
where did the "l" come from?

Reply:
its what is used in the NXP HAL.  hs_lspi/hs_spi seem to be intermixed.

Reply:
https://github.com/zephyrproject-rtos/hal_nxp/blob/87850e41cd48e4472adbae03c9623ea349255d48/mcux/devices/LPC55S69/fsl_clock.h#L258